com.unity.multiplayer.mlapi/Runtime/Core/NetworkBehaviour.cs

```diff
@@ -486,7 +489,7 @@ internal void InitializeVariables()
         private static ProfilerMarker s_NetworkBehaviourUpdate = new ProfilerMarker($"{nameof(NetworkBehaviour)}.{nameof(NetworkBehaviourUpdate)}");
 #endif
 
-        internal static void NetworkBehaviourUpdate()
+        internal static void NetworkBehaviourUpdate(SnapshotSystem snapshot)
```

Comment:
how can this PR be a "chore"?

Reply:
Yeah, I guess I have difficulties with the conventional commits classification. What would you have seen those as ?

Feat: ? It's not one yet. But I guess that's the one that fits best?
Fix: ? Nope
Test: ? Nope
Groundwork: I wish it existed
Redesign: Doesn't exist either

Reply:
Also, doing a series of rebase and amend doesn't seem too productive. I'll make sure I squash this PR into a single "feat: " commit at the end. Is this something that would work for you?

Reply:
I believe this is a "feat: " and I feel like I don't have enough context to review this PR.